Code/Tools/SceneAPI/SceneCore/Components/SceneSystemComponent.h

```diff
@@ -19,7 +19,7 @@ namespace AZ
         {
             // Scene system components are components that can be used to create system components
             //      in situations where the full initialization and/or construction of regular
-            //      system components don't apply such as in the Project Configurator's advanced
+            //      system components don't apply such as in the Project Manager's advanced
```

Comment:
There is no parity for this in Project Manager

Reply:
Thanks will remove this mention